src/linagora.esn.calendar/app/routes.js

```diff
@@ -34,19 +34,12 @@ function routesConfig($stateProvider) {
     })
     .state('calendar.main.participation', {
       url: '/participation',
-      resolve: {
-        event: function($log, $state, $location, notificationFactory, calEventService) {
-          const { jwt, eventUid } = $location.search();
+      onEnter: function($state, $location, calPartstatJWTService) {
```

Comment:
why using onEnter instead of resolve?

Reply:
From https://github.com/angular-ui/ui-router/wiki:

> You can use `resolve` to provide your controller with content or data that is custom to the state. resolve is an optional map of dependencies which should be injected into the controller.

> There are also optional `onEnter` and `onExit` callbacks that get called when a state becomes active and inactive respectively. The callbacks also have access to all the resolved dependencies.

As in this case, we don't provide any controller with any data, so using `resolve` is inappropriate here I think.